models/gas/gas_fees_schema.yml

```diff
@@ -0,0 +1,70 @@
+version: 2
+
+models:
+  - name: gas_fees
```

Comment:
I see several columns here that seem to me to be evm- and/or eth-specific (e.g. does solana have a concept similar to gas limit? do other evm chains implement a burning mechanism like eip-1559?).

If all these columns remain relevant to all other chains, all my points below are moot.

Otherwise: I think we have an interesting design decision here:
- do we want to make this table fully encompassing of all different chains (like dex.trades does with protocols)?
- or do we want to have more dedicated views (e.g. gas-evm, gas-solana) to cover nuances of the protocol and have a more "abstracted out" perspective of all chains?

For this I would ask: what kind of analyses do we envision will happen cross-chain (particularly between diff technologies like evm v solana)

Reply:
So maybe it's even more obvious in the Avalanche PR here: #1751 but 
- Yes Solana and other EVM compatible blockchains share some (not all) properties and post EIP 1559 mechanisms. 
- I think a good compromise is too include all columns that can be shared across chains, but leave specific/interesting columns in individual models, and then expose both

Reply:
> I think a good compromise is too include all columns that can be shared across chains, but leave specific/interesting columns in individual models, and then expose both

Agree! 

we could have the structure be like:
- gas.fees
- gas.ethereum_fees
- gas.avalanche_fees
- gas.solana_fees
- (etc)

as for naming convention for columns in gas.fees we could:
- choose a generic name
- stick to evm names and explain mapping in schema file (=>documentation)
- _other ideas?_

Personally I gravitate to the second, since evm chains and associated concepts are pretty dominant in the web3 space, I would be surprised if users were confused by evm naming.

Reply:
Yes exactly, if you look at my Avalanche PR this is exactly what I've done there: #1751
Side note: I actually don't think Solana will work atm, too many transactions = too much compute/time spent time generate this spell. Will focus on shipping the three EVM compatible L1s (Ethereum, BNB, Avalanche) + blog post and then I'll work on Arbitrum/Optimism/Gnosis for L2s